packages/subgraph/CHANGELOG.md

```diff
@@ -8,18 +8,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [UPCOMING]
 
 ### Changed
+
 - Changes callId in `Executed` event from `uint256` to `bytes32`
 
 ## 0.6.2-alpha
 
 ### Added
 
+- Added the `MembershipContractAnnounced` event.
```

Comment:
move it into the section `## [UPCOMING]`

Reply:
Fixed in efbd5f72a89f102043c4a983c94d439c70bbe103